### DISCUSSION TITLE
Todoist Integration for Pomatez?

### DISCUSSION THREAD

**Discussion by nirmalhk7** [zidoro/pomatez#139] [Ideas]
Since Pomatez in itself is a laptop-only app, why not make the todo-list device-independent by providing integrations with Todoist?

**Comment by roldanjr** [comment]
Yes, that's a really cool idea. I have a lot of plans to improve this tiny app.

But as of now. I would like to ask for apology cause I really don't have that much time to do that.

Hoping by the end of this year I can clean up the codebase to be more open source friendly, so that anyone can easily contribute. 

Thanks for your idea @nirmalhk7 .

**Comment by sekwah41** [comment]
This has also been raised as #210. I use Todoist for my daily tasks so I may take a look into doing this at some point. Though there are a fair few other issues to focus on first.
